src/Flowers.js

```diff
@@ -1,13 +1,19 @@
 export function Flowers() {
-    return [<h3>SILAS MARNER</h3>,
-    <p>An unfussy rose of soft mid pink, the petals are pale on the reverse and fade at the edges creating a gentle quality.</p>,
-    <img src="C:\Users\Dell\Desktop\my--first-react-app\public\images\SILAS MARNER.jpg" alt="SILAS MARNER"></img>,
-    <h3>GABRIEL OAK</h3>,
-    <p>A magnificent variety, bearing large, many petalled rosette blooms. They are a striking shade of deep pink, the outer petals of each bloom paling slightly over time. The beauty of the blooms is enhanced by a wonderful, strong fruity fragrance. </p>,
-
-    <img src="C:\Users\Dell\Desktop\my--first-react-app\public\images\GABRIEL OAK.jpg" alt="GABRIEL OAK"> </img> ,
-    <h3>HARLOW CARR</h3>,
-    <p>Bears flowers of the most perfect formation – shallow cups of the purest mid pink. They flower very freely from early summer until well into the autumn and have a strong, pure Old Rose fragrance. </p>,
-    <img src="C:\Users\Dell\Desktop\my--first-react-app\public\images\HARLOW CARR.jpg" alt="HARLOW CARR.jpg"></img>]
+    return <>
+        <h3>SILAS MARNER</h3>
```

Comment:
You don't need to hardecode any value here 
the value of `h3` should come from the `props` 

you need to pass this information to your component to make re-usable